SOURCES/xen_dom0_64bit-efi.patch

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
If the patch comes unmodified from an upstream commit, then it's better to provide it in the form of a patch created using git format-patch (or downloaded from a web frontend to the git repository, in raw text format including git commit headers). You can add comments on top of all this, if necessary.

If you had to modify the patch, then the best is to still start with the full patch with headers from upstream, modify it, and add comments on top explaining what you had to change.

Note: you don't need to credit yourself in the patch, as you will be credited already both through this repository's git commit history and in the changelog of the RPM.

Reply:
I see that there are two upstream commits. If they can be applied as is, then the best is probably to create one patch per commit and to apply both. The closer the patches are to upstream, the better.